less/components/footer.less

```diff
@@ -101,8 +101,8 @@ footer a {
 
 .block_make_submission_link {
 	.button();
-	border: 1px solid @black;
-	background-color: @black;
+	border: 1px solid #000
+	background-color: #000
```

Comment:
and missing `;` here

Reply:
oh embarrassing mistakes.. :[

Reply:
(Actually, @NateWr I think it’s because I was finding and replacing all mentions of the variable `@black` with `#000`, many of which were appearing mid-line in a value and didn’t properly look over all the changes! My bad)